mx_javacompliance.py

```diff
@@ -213,9 +206,19 @@ def __cmp__(self, other):
         return mx.compare(self._parts, other._parts)
 
     def __contains__(self, other):
+        """
+        :param other: JavaCompliance instance that represents one specific version
+        :return: if `other` is satisfied by `self`
+        """
         if isinstance(other, (int, str)):
             other = JavaCompliance(other)
-        assert other._high_bound() is not None, "Contains check cannot be done with version ranges"
+        assert (
+                len(other._parts) == 1 and
+                (
+                        other._high_bound() is None or
+                        other._high_bound() == other.value
```

Comment:
How can this happen? Can you please add extra tests to `_test()` in `mx_javacompliance.py` to cover this case as well as any other cases made relevant by your changes.

Reply:
Sure, I'll add tests for range checks.